readit/cli.py

```diff
@@ -49,17 +50,16 @@ def main(insert, add, tag, delete, clear,
                 validate_url = requests.get(url)
                 validate_code = validate_url.status_code
                 if validate_code == 200:
-                    database_connection.add_url(url)
+                    result(database_connection.add_url(url))
```

Comment:
@bhavin192 
As you suggested to return **True/False** from methods. Here messages returned from methods like ```add_url``` or ```delete_url``` are different. So that we need to return message with True or False flag.

Reply:
The message should be decided by the calling function not by the method of class, the method from class should just tell you if the operation is successful or not.